samples/matter/common/src/bridge/ble_connectivity_manager.cpp

```diff
@@ -159,6 +161,13 @@ void BLEConnectivityManager::DisconnectionHandler(bt_conn *conn, uint8_t reason)
 	bt_addr_le_to_str(bt_conn_get_dst(conn), addrStr, sizeof(addrStr));
 	LOG_INF("Disconnected: %s (reason %u)", addrStr, reason);
 
+#ifdef CONFIG_BT_SMP
+	int ret = bt_unpair(BT_ID_DEFAULT, bt_conn_get_dst(conn));
```

Comment:
It's definitely not a good idea to unpair on disconnect. It will result in starting pairing again on every re-connect or reboot and this is not what we want to do.
I think we should only unpair the device if it was permanently removed from the bridge (not temporarily disconnected).

Reply:
Ok, repaired.